src/test/java/org/broadinstitute/consent/http/db/DatasetDAOTest.java

```diff
@@ -145,6 +147,131 @@ public void testFindNeedsApprovalDataSetByDataSetId() {
         assertTrue(datasets.get(0).getNeedsApproval());
     }
 
+    @Test
+    public void testGetNIHInstitutionalFile() {
+        Dataset dataset = createDataset();
+
+        createFileStorageObject(
+                dataset.getDataSetId().toString(),
+                FileCategory.ALTERNATIVE_DATA_SHARING_PLAN
+        );
+        FileStorageObject nihFile = createFileStorageObject(
+                dataset.getDataSetId().toString(),
+                FileCategory.NIH_INSTITUTIONAL_CERTIFICATION
+        );
+        createFileStorageObject();
+        createFileStorageObject(
+                dataset.getDataSetId().toString(),
+                FileCategory.DATA_USE_LETTER
+        );
```

Comment:
Here and again on line 154, it would be nice to have some commentary about why these creates are here. Additionally, if we are creating these for some reason, we should have some asserts that ensure what it is that we do not want to see, i.e., there should be no data use letter file object on the dataset in this case.